README.md

```diff
@@ -1,2 +1,18 @@
-# interoperability-placeholder
+# Interoperability
 This is a placeholder for 2023 interoperability project 
+
+## Get Started
+
+1. Run `./startup.sh init` to start-up the docker containers on the first run or after calling `./startup.sh destroy`. Use `./startup.sh up` for subsequent runs after calling `init` without calling `destory`.
+
+1. The default User username for OpenHIM is `interop@openhim.org` and password is `interop-password`. The default Client username is `interop-client` and password is `interop-password`.
+
+1. `./startup.sh down` or `./startup.sh destroy` to shutdown the servers.
+
+1. Visit https://localhost:9000 and login with the following credentials email: `interop@openhim.org` and password: `interop-password`.
+
+1. Once logged in, visit https://localhost:9000/#!/mediators and select the only mediator with the `Name` 'mediator'.
```

Comment:
I'm seeing a mediator with the `Name` 'Loss to Follow Up Mediator'

Reply:
![Captura de pantalla 2023-01-13 a la(s) 14 22 43](https://user-images.githubusercontent.com/21312057/212380798-95bc2aa7-aadb-4838-ac5d-6b0b14dd1965.png)